dev/tasks/dev/utils/inquiries.js

```diff
@@ -68,10 +68,21 @@ module.exports = {
 		return new Promise( ( resolve ) => {
 			inquirer.prompt( [ {
 				name: 'description',
-				message: 'Package description (one sentence):'
+				message: 'Package description (one sentence, must end with period):'
 			} ], ( answers ) => {
-				resolve( answers.description || '' );
+				resolve( appendPeriodIfMissing( answers.description ) );
 			} );
 		} );
 	}
 };
+
+function appendPeriodIfMissing( text ) {
+	text = text ? String.prototype.trim.call( text ) : '';
```

Comment:
Do you know any real case in which argument of that function may be a number? I don't see any. It's used in one place and its parameter is called "text". So KISS.

If there was any possibility that `answers.description` could be a number, we should've fixed it there, not in this function.